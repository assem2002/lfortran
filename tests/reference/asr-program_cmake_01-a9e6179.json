{
    "basename": "asr-program_cmake_01-a9e6179",
    "cmd": "lfortran --show-asr --no-color {infile} -o {outfile}",
    "infile": "tests/../integration_tests/program_cmake_01.f90",
    "infile_hash": "0e854d9b2ed613d6193d78290dc20a45ec66f60ab17e1c0229c3febe",
    "outfile": null,
    "outfile_hash": null,
    "stdout": "asr-program_cmake_01-a9e6179.stdout",
    "stdout_hash": "c5670245b884464cb8ab5c3857099e8ebc05f1079fe7fda2a4c8a3de",
    "stderr": null,
    "stderr_hash": null,
    "returncode": 0
}