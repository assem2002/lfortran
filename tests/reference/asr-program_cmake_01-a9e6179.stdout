(TranslationUnit
    (SymbolTable
        1
        {
            testfortran:
                (Program
                    (SymbolTable
                        2
                        {
                            
                        })
                    testfortran
                    []
                    [(Print
                        [(StringFormat
                            ()
                            [(StringConstant
                                "Hello"
                                (Character 1 5 ())
                            )]
                            FormatFortran
                            (Character -1 0 ())
                            ()
                        )]
                        ()
                        ()
                    )]
                )
        })
    []
)
